lib/cluster/state.ts

```diff
@@ -218,41 +285,49 @@ class State {
   /**
    * Removes a room in the full state, for a given node.
    *
-   * @param  {string} roomId
-   * @param  {string} nodeId
+   * @param roomId
+   * @param nodeId
    */
-  removeRealtimeRoom (roomId, nodeId) {
+  removeRealtimeRoom (roomId: string, nodeId: string) {
     const room = this.realtime.get(roomId);
 
     /**
      * If a local room gets deleted, there aren't sync data about it
+     * @todo investigate this
      */
-    if (!room) {
+    if (! room) {
       return;
     }
 
     room.removeNode(nodeId);
 
-    if (!room.hasNodes()) {
+    if (! room.hasNodes()) {
       this.realtime.delete(roomId);
+
       global.kuzzle.koncorde.remove(
         roomId,
         toKoncordeIndex(room.index, room.collection));
     }
   }
 
-  countRealtimeSubscriptions (roomId) {
+  countRealtimeSubscriptions (roomId: string): number {
     const room = this.realtime.get(roomId);
 
-    if (!room) {
+    if (! room) {
       return 0;
     }
 
     return room.countSubscriptions();
   }
 
   listRealtimeRooms () {
-    const list = {};
+    const list: {
```

Comment:
Why not writing an interface for this? So we could annotate the return type with it?